core-bundle/src/Controller/BackendController.php

```diff
@@ -107,7 +108,7 @@ public function previewAction(Request $request): Response
 
         $this->get('contao.framework')->initialize();
 
-        $controller = new BackendPreview();
+        $controller = new FrontendIndex();
```

Comment:
Is this just for BC reason? Then we should make sure the route is no longer used and add a deprecation warning?

Reply:
I addressed this